proto/py/compute_image_tools_proto/inspect_pb2.pyi

```diff
@@ -0,0 +1,151 @@
+# @generated by generate_proto_mypy_stubs.py.  Do not edit!
```

Comment:
Copyright?

Reply:
I was following this guidance: "License headers are generally not required for generated code..." from go/releasing/preparing